geometry/frame_test.cpp

```diff
@@ -28,7 +44,7 @@ TEST_F(FrameDeathTest, SerializationError) {
     serialization::Frame message;
     World1::WriteToMessage(&message);
     World2::ReadFromMessage(message);
-  }, "tag ==");
+  }, "\\(tag\\) ==");
```

Comment:
`R"(\(tag\) ==)"`

Reply:
Done, but I'm not sure if the extra parentheses are better than the extra `\`.